bin/build.sh

```diff
@@ -0,0 +1,10 @@
+for example_file in exercises/**/*.example
+do
+  exercise_dir=$(dirname $example_file)
+  exercise=$(basename $example_file .example)
+  mv "$exercise_dir/$exercise.elm" "$exercise_dir/$exercise.impl"
+  mv "$exercise_dir/$exercise.example" "$exercise_dir/$exercise.elm"
+  elm-test exercises/**/*Tests.elm
+  mv "$exercise_dir/$exercise.elm" "$exercise_dir/$exercise.example"
+  mv "$exercise_dir/$exercise.impl" "$exercise_dir/$exercise.elm"
```

Comment:
Oh thanks, this whole copying around business seems hacky but it looks like the Haskell track is similar. Ugg